pkg/agent/baker.go

```diff
@@ -419,7 +419,7 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 			return cs.Properties.OrchestratorProfile.IsNoneCNI()
 		},
 		"IsMariner": func() bool {
```

Comment:
don't looove this, but it works...would be nice to base this off distro instead?